packages/app/package.json

```diff
@@ -24,53 +24,53 @@
     "lint": "backstage-cli package lint"
   },
   "dependencies": {
-    "@backstage/app-defaults": "^1.4.4",
+    "@backstage/app-defaults": "^1.4.7",
     "@backstage/catalog-model": "^1.4.3",
-    "@backstage/core-app-api": "^1.11.0",
-    "@backstage/core-components": "^0.13.6",
-    "@backstage/core-plugin-api": "^1.7.0",
-    "@backstage/integration-react": "^1.1.20",
-    "@backstage/plugin-api-docs": "^0.9.12",
-    "@backstage/plugin-catalog": "^1.14.0",
-    "@backstage/plugin-catalog-common": "^1.0.17",
-    "@backstage/plugin-catalog-graph": "^0.2.37",
-    "@backstage/plugin-catalog-import": "^0.10.1",
-    "@backstage/plugin-catalog-react": "^1.8.5",
-    "@backstage/plugin-github-actions": "^0.6.6",
-    "@backstage/plugin-org": "^0.6.15",
-    "@backstage/plugin-permission-react": "^0.4.16",
-    "@backstage/plugin-scaffolder": "^1.15.1",
-    "@backstage/plugin-search": "^1.4.1",
-    "@backstage/plugin-search-react": "^1.7.1",
-    "@backstage/plugin-tech-radar": "^0.6.9",
-    "@backstage/plugin-techdocs": "^1.8.0",
-    "@backstage/plugin-techdocs-module-addons-contrib": "^1.1.1",
-    "@backstage/plugin-techdocs-react": "^1.1.12",
-    "@backstage/plugin-user-settings": "^0.7.11",
-    "@backstage/theme": "^0.4.3",
+    "@backstage/core-app-api": "^1.11.3",
```

Comment:
I wonder if we should consider pinning these dependencies like we do in backstage-showcase?  Maybe not on this PR necessarily but could be worth a discussion.